R/data.R

```diff
@@ -0,0 +1,18 @@
+#' Ebola linelist data from Fang et al. (2016)
+#'
+#' Linelist data for the Ebola virus collected in Sierra Leone.
+#'
+#' @format A `tibble` with 8,358 rows and 8 columns:
+#' \describe{
+#'   \item{id}{Unique identification number for the case}
+#'   \item{name}{Name as character, omitted}
+#'   \item{age}{Age as numeric}
+#'   \item{sex}{Sex as character, either "F", "M" or NA}
+#'   \item{date_of_symptom_onset}{The date symptoms began}
+#'   \item{date_of_sample_tested}{The date the sample was tested}
+#'   \item{district}{The district (ADM2)}
+#'   \item{chiefdom}{The chiefdom (ADM3)}
+#' }
+#' @family data
+#' @source <https://www.pnas.org/doi/full/10.1073/pnas.1518587113>
+"ebola"
```

Comment:
I think we might want to be a little more verbose here i.e `sierra_leone_ebola_outbreak_data` or similar?

Reply:
How do people know that just from looking? It could be a function

Reply:
I very much don't think it should just be ebola - location context is important

Reply:
Yes that's fair r.e. it could be a function